stored_queries/GO/GO_get_hierarchicalAncestors.yaml

```diff
@@ -22,21 +22,30 @@ params:
         default: 0
         description: Result offset for pagination
         maximum: 100000
-query_prefix: WITH GO_test_term 
+    ts:
+      type: integer
+      title: Versioning timestamp
+query_prefix: WITH GO_terms
 query: |
-  LET term_id = CONCAT("GO_test_term/", @key)
-  LET results1 = (
-    FOR v_relationship, e_relationship IN 1..1000000 OUTBOUND term_id GO_test_edges_relationship
-      FILTER e_relationship.relationship_type == 'part_of' 
-        OR e_relationship.relationship_type == 'has_part'
-        OR e_relationship.relationship_type == 'occurs_in'
-      RETURN {term: v_relationship, edge: e_relationship}
-  )
-  LET results2 = (
-    FOR v_isa, e_isa IN 1..1000000 OUTBOUND term_id GO_test_edges_isa 
-      RETURN {term: v_isa, edge: e_isa}
-  )
-  FOR x IN UNION(results1, results2)
-    SORT x._key ASC
-    LIMIT @offset, @limit
-    RETURN DISTINCT x
+  FOR t in GO_terms
+    FILTER t.id == @id
+    FILTER t.created <= @ts AND t.expired >= @ts
+    limit 1
+    LET results1 = (
+      FOR v_relationship, e_relationship IN 1..100 OUTBOUND t GO_edges_relationship
+        FILTER e_relationship.relationship_type == 'part_of' 
+          OR e_relationship.relationship_type == 'has_part'
+          OR e_relationship.relationship_type == 'occurs_in'
+        FILTER e_relationship.created <= @ts AND e_relationship.expired >= @ts
+        RETURN {term: v_relationship, edge: e_relationship}
+    )
+    LET results2 = (
```

Comment:
You can remove this `results2` subquery and add an extra condition in the above query for `OR e_relationship.type == 'is_a'`